### ISSUE TITLE
Variations: Selected Attributes and Options Screen

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#3183]
Partially depends on #3182. 

Refs p91TBi-3Oc-p2, p91TBi-2Ur-p2.

After picking an attribute and the options to offer (#3182), the user will be sent to this screen. 

## Design

<img src="https://user-images.githubusercontent.com/198826/99757383-7b853100-2aac-11eb-8b9d-95dd8611c3a2.png" width="500">

## Goals

- [ ] Present the selected attributes and options (attribute values) that will be used to generate the variations. The generation will be done in a separate task. 
- [ ] Tapping on the "Add Attribute" button will present the attribute selection (#3181) to values selection (#3182) flow again. 
- [ ] Activating a row will present the values management screen (#3182), with the options pre-selected. 

---

Android: https://github.com/woocommerce/woocommerce-android/issues/3209

**Comment by Ecarrion** [comment]
Closed in favor of #3035, #3036 and #3037
